lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const ManifestGatherer = require('./manifest');
+
+class StartUrl extends ManifestGatherer {
+  afterPass(options) {
+    let startUrl = options.url;
+
+    return super.afterPass(options)
+      .then(manifest => {
+        if (!manifest) {
+          return Promise.reject(`Unable to retrieve manifest at ${options.url}`);
+        }
+
+        startUrl = manifest.value.start_url.raw;
```

Comment:
I believe we want `manifest.value.start_url.value` here and then need to reject if there isn't a `value`. 

We may also want to surface the `manifest.value.start_url.debugString` (e.g. if it's an invalid URL or cross-origin) and warn that Lighthouse is testing the default URL (`options.url` in this case) instead.

I wonder if we should show which `start_url` we tested regardless, actually, since that could be really annoying if you have an error in your `start_url` and so we use the default instead, or we're calculating the url relative to the document URL differently than the developer expects